Session-4/GenericStack/Stack.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace GenericStack
+{
+  internal class Stack<T>
+  {
+    private List<T> _data = (List<T>) null;
+
+    public Stack() => this._data = new List<T>();
+
+    public void PushElement(T item) => this._data.Add(item);
+
+    public void PopElement() => this._data.RemoveAt(this._data.Count<T>() - 1);
```

Comment:
Wrap the code in braces and rename the method as Pop.
Also what if stack is empty, you need to throw exception.